app/controllers/api/oceans_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Api::OceansController < ApplicationController
-  def index
-    oceans = Array.new(4) do
+  def index(n_oceans = 4)
```

Comment:
You are a little bit on the wrong track here: 
http://guides.rubyonrails.org/action_controller_overview.html#parameters